packages/channels/botpress-channel-web/src/views/web/convo/index.jsx

```diff
@@ -127,32 +130,32 @@ export default class Convo extends React.Component {
   renderWelcomeMessage() {
     const message = this.props.config.welcomeMsgText || DEFAULT_WELCOME_MESSAGE
 
-    return <div className={style.paragraph}>{message}</div>
+    return <div className={`${style.paragraph} botpress__paragraph`}>{message}</div>
   }
 
   renderHeader() {
     return (
       <div
-        className={style.header}
+        className={`${style.header} botpress__header`}
         style={{
           color: this.props.config.textColorOnForeground,
           backgroundColor: this.props.config.foregroundColor
         }}
       >
-        <div className={style.line}>
-          <div className={style.title}>
+        <div className={`${style.line} botpress__line`}>
+          <div className={`${style.title} botpress__title`}>
             {this.renderAvatar()}
             {this.renderName()}
           </div>
         </div>
-        <div className={style.text}>{this.renderWelcomeMessage()}</div>
+        <div className={`${style.text} botpress__text`}>{this.renderWelcomeMessage()}</div>
       </div>
     )
   }
 
   renderComposer() {
     return (
-      <div className={classnames(style.composer)}>
+      <div className={`${classnames(style.composer)} botpress__composer`}>
```

Comment:
_composer_ is some internal slang here, something like "new-message-container" sounds more appropriate, no?